### DISCUSSION TITLE
Adding CORS headers to Laravel Valet | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#583] [General]
# Adding CORS headers to Laravel Valet | Rias.be

When working on a project with subdomains, you can run into CORS issues locally when using Laravel Valet when your assets are on the main domain.

https://rias.be/blog/adding-cors-headers-to-laravel-valet

**Comment by MaximVanhove** [comment]
Thank you, it worked flawlessly! 😊
